v2/CHANGES

```diff
@@ -88,7 +88,7 @@ To port encodings forward:
 Package changes:
 - [ ] api/transport -> v2
 - [ ] api/peer -> v2
-- [x] api/middleware -> v2/yarpcmiddlware
+- [x] api/middleware -> v2
```

Comment:
This is intended to be a template, so the x is unnecessary. Perhaps we should just get rid of the checkboxes except in the top couple sections that actually are TODO for the branch.